packages/application-shell/package.json

```diff
@@ -18,11 +18,21 @@
   "publishConfig": {
     "access": "public"
   },
-  "main": "./src/index.js",
-  "module": "./src/index.js",
-  "engines": {
-    "node": ">=8",
-    "npm": ">=5"
+  "main": "dist/application-shell-index.cjs.js",
+  "module": "dist/application-shell-index.es.js",
+  "files": [
+    "dist",
+    "package.json",
+    "LICENSE",
+    "README.md"
+  ],
+  "scripts": {
+    "prebuild": "rimraf dist/**",
+    "build": "npm run build:es && npm run build:cjs",
+    "build:cjs": "cross-env NODE_ENV=production rollup -c ../../rollup.config.js  -f cjs ./index.js --dir ./dist --chunkFileNames application-shell-[name]-[hash].cjs.js --entryFileNames application-shell-[name].cjs.js --experimentalCodeSplitting",
+    "build:es": "cross-env NODE_ENV=production rollup -c ../../rollup.config.js  -f es ./index.js --dir ./dist --chunkFileNames application-shell-[name]-[hash].es.js --entryFileNames application-shell-[name].es.js --experimentalCodeSplitting",
+    "build:es:watch": "npm run build:es -- -w",
+    "test:build:es": "cross-env NODE_ENV=development rollup -c ../../rollup.config.js  -f es ./src/test-utils.js --o ./dist/test-utils/index.js"
```

Comment:
Is this used somewhere? 🤔

Reply:
@dferber90 and I decided that it didn't make sense to add `src/test-utils.js` to the main export. Now users will access it in their code like 
```import testUtils from '@commercetools-frontend/applicatoin-shell/test-utils```

Reply:
Ah, but it should be added to the main build as well. WOops

Reply:
This is the same approach as `react-apollo` uses btw.

Reply:
> Ah, but it should be added to the main build as well

Yeah I meant that 👌